charmbits/service/service.go

```diff
@@ -123,22 +122,10 @@ func (svc *Service) Stop() error {
 }
 
 // Started reports whether the service has been started.
-func (svc *Service) Started() bool {
+func (svc *Service) Started() (bool, error) {
 	return svc.osService(nil).Running()
 }
 
-// StopAndRemove stops and removes the service completely.
-func (svc *Service) StopAndRemove() error {
```

Comment:
We can't just remove this functionality entirely.
Rather than break the API, let's please reimplement it in terms
of what we now have (we'll need to add a Remove method to OSService).

We can consider later whether we might want to drop Stop and Remove in favour of
just "Remove".